src/devices/Shtc3/Measure.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Units;
+
+namespace Iot.Device.Shtc3
+{
+    /// <summary>
+    /// Shtc3 measure
+    /// </summary>
+    public readonly struct Measure
```

Comment:
- humidty arg should contain units in name and description (i.e. relativeHumidity), note that name relativeHumidity implies return value from 0..1.
- Please call this Shtc3Measurement or have an overload like `bool TryRead(out Temperature temperature, out double relativeHumidity)` (we currently use that instead of separate struct in many places)